### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
part of #8581

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/index.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ The following operating systems are supported:
 
 [HTTP.sys server](xref:fundamentals/servers/httpsys) (formerly called [WebListener](xref:fundamentals/servers/weblistener)) doesn't work in a reverse proxy configuration with IIS. Use the [Kestrel server](xref:fundamentals/servers/kestrel).
 
+For information on hosting to Azure, see  [Host ASP.NET Core on Azure App Service](xref:host-and-deploy/azure-apps).
```
Comment (guardrex) [inline aspnetcore/host-and-deploy/iis/index.md]: * I'd recommend "in" over "to."
* Can use just the xref ... we can suck the title thru on this.
* There's an extra space before the link.

> For information on hosting in Azure, see \<xref:host-and-deploy/azure-apps>.

Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/index.md]: ... oh ... and I added "index" to the end. That's a landing page of a node.

Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/iis/index.md]: Remind me of direct xref syntax - or better yet edit in  browser and make a commit to fix.

Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/index.md]: That's a cut-and-paste :point_up:.

I'll do it. Stand-by.

**Pull request merged**